open-metadata-resources/open-metadata-deployment/charts/odpi-egeria-lab/templates/egeria-ui.yaml

```diff
@@ -72,6 +72,10 @@ spec:
           envFrom:
             - configMapRef:
                 name: {{ include "myapp.fullname" . }}-configmap
+          command:
+            - "/bin/bash"
+            - "-c"
+            - "java -jar /deployments/user-interface/ui-chassis-spring-*.jar --server.ssl.trust-store=/deployments/truststore.p12"
```

Comment:
the UBI8 openjdk image is configured differently.... We set JAVA_APP_JAR to the application that is to be run. We do not set a command to be executed. This follows the standard pattern for that image

Reply:
There's a little more on the UBI image captured in the issue originally used to create it https://github.com/odpi/egeria/issues/3580

Reply:
The way the environment is setup also means the ssl parm shouldn't be needed - at least to use default certs (we may still have work to do to inject custom certs into the deployment)